<?xml version="1.0"?>
<launch>
    <!-- Follower -->
    <node name="path_follower" pkg="igvc_navigation" type="path_follower" output="screen" >
        <!-- Maxmimum wheel velocity allowed before stopping motors -->
        <param name="maximum_vel" type="double" value="2"/>
        <param name="axle_length" type="double" value="0.52"/>
        <!-- Distance along the path to generate smooth control law for  -->
        <param name="lookahead_dist" type="double" value="0.4"/>
        <!-- Frequency (hz) at which simulation is done -->
        <param name="simulation_frequency" type="double" value="10.0"/>
        <!-- How many seconds in the future should be simulated-->
        <param name="simulation_horizon" type="double" value="5.0"/>
        <param name="target_reached_distance" type="double" value="0.1"/>
        <param name="target_move_threshold" type="double" value="0.1"/>
        <!-- How far away robot should be from the waypoint before it stops -->
        <param name="stop_dist" type="double" value="0.7"/>
        <!-- Smooth Control Parameters -->
        <param name="target_v" type="double" value="0.4"/>
        <param name="k1" type="double" value="1.8"/>
        <param name="k2" type="double" value="2"/>
        <param name="debug" type="bool" value="true"/>
    </node>
</launch>